src/java/fr/paris/lutece/portal/service/admin/DefaultImportAdminUserService.java

```diff
@@ -247,9 +247,9 @@ private int getStatus( String strStatus, String strLastName, String strFirstName
     
     private void readAttribute( String strValue, List<String> listAdminRights, List<String> listAdminRoles, List<String> listAdminWorkgroups,  Map<Integer, List<String>> mapAttributesValues )
     {
-        if ( StringUtils.isNotBlank( strValue ) && ( strValue.indexOf( getAttributesSeparator( ) ) > 0 ) )
+        int nSeparatorIndex = strValue.indexOf( getAttributesSeparator( ) );
```

Comment:
What if `strValue` is `null` ? Previously, `StringUtils.isNotBlank` would check for this

Reply:
Will be corrected